### DISCUSSION TITLE
winキーが常に押され, 正常にキーを押すことができない.

### DISCUSSION THREAD

**Discussion by lieeeeeeeee** [Yowkees/keyball#584] [Q&A]
# 不具合状況
あるキーのみを押すと "win + (あるキー)" を押したときと同じような状況になります.
これを改善するためには "win + alt + ctrl" を数秒同時に押すことで改善されます.
しかし次に `enter` と `backspace` のキーが機能しなくなります.
またこれらの現象は同じPCに接続している, 別のキーボードでも同様に発生します(keyballを接続すると).
キーロック機能はすべてOFFに設定してあります.

# 試したこと
別のキーボードで発生する問題はkeyballの接続を切断すると改善されます.
キーボードレイアウトを "Microsoft IME" から "Google 日本語入力" に変更しても同様に発生します.
別のPCにkeyballを接続しても同様に発生します.
ファームウェアをリセットし書き換えても変化はありませんでした.
左右どちらのkeyballに接続しても変化はありませんでした.
winキーを単体で押したとき, 特にエラーなく画面左から通常通りポップアップが表示されます.

# 環境
device: keyball39
os: windows 11
firmware: Keyball39_via_v1.3.2

**Comment by apio1969** [comment]
皆さんそれぞれキーマップが違うと思いますので、どのキーの組み合わせなのかが文章だけでは読み取りできないですが、
どこかしかの半田不良の可能性がありそうですね。
よければ、基盤の半田状況の写真をUpされてはいかがでしょうか？
それをみれば、どなたか有識者からコメントをいただけるあかと思います。

**Reply by lieeeeeeeee** [reply]
ご返事ありがとうございます.
キーマップは変更しておらず, ファームウェアを書き込み後, 編集は全く行っていない環境で発生している問題となっております.

半田不良の可能性があるご指摘ありがとうございます.
再度私の方で確認をさせていただき, そのうえで不具合が解消されなかった場合は, 写真を添付したうえで加えてご質問させていただきます.
ありがとうございました!
